examples/ssx-test-token-gated/README.md

```diff
@@ -0,0 +1,338 @@
+# SSX ENS Token Gated + RainbowKit
+
+This example will show developers how to enable token-gated access in their dapp with SSX. Additionally, it will show a developer how to integrate and use RainbowKit in an SSX-powered dapp.
+
```

Comment:
Can we add instructions here: 

To run the completed example: 
- Add your Alchemy API key to `.env` (see below)
- Once SSX packages are installed, run `yarn install` and `yarn start` in this example directory.

Read on to learn how to build the example from our `create-ssx-dapp` example.